.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
+
 # Keep environment variables out of version control
-.env
-.env.test
-.wwebjs_cache/*
```

Comment:
what's happened to wwebjs_cache?

Reply:
we are not using whatsapp-web package anymore so it is not relavnt